scripts/tool/convert_image_to_data_url.py

```diff
@@ -5,6 +5,9 @@
 from PIL import Image
```

Comment:
Please try to run this script on macos and ubuntu to make sure the script is robust.

Reply:
![image](https://github.com/microsoft/promptflow/assets/95729303/e65bd483-e8b3-4623-a651-11ba1eae9262)
Tested in ubuntu, it works fine, thanks.

Reply:
Tested in devbox, it works fine, thanks.

Reply:
We have `ubuntu` and `macos` CI env now, which we can use to run gen package tool meta tests on different os. For example,

https://github.com/microsoft/promptflow/blob/9724c4ae87993cf885c434511e68a9cc33e8585a/.github/workflows/promptflow-global-config-test.yml#L40

@dorisjoy to plan the work.

Reply:
Tested on macos, thanks.
https://github.com/microsoft/promptflow/actions/runs/6481366108/job/17598709739?pr=685
![image](https://github.com/microsoft/promptflow/assets/95729303/2c56065e-0a4d-49e4-a6f4-a805c00d261d)
![image](https://github.com/microsoft/promptflow/assets/95729303/cc1cc6bb-adc0-4a0b-a891-d5383e1ec0b1)